assets/app/scripts/controllers/util/error.js

```diff
@@ -31,4 +31,9 @@ angular.module('openshiftConsole')
     if (params.error_description) {
       $scope.errorDetails = params.error_description;
     }
+    
+    $scope.reloadConsole = function() {
```

Comment:
Should the back to console link on the error page always force a full reload?  Today it just navigates back to the main page but in the case of API discovery failures we need a full reload to re-run discovery.